Easydict/objc/ViewController/Window/BaseQueryWindow/EZBaseQueryViewController.m

```diff
@@ -498,8 +531,11 @@ - (void)startOCRImage:(NSImage *)image actionType:(EZActionType)actionType {
             [self updateQueryTextAndParagraphStyle:inputText actionType:actionType];
             
             if (error) {
-                NSString *errorMsg = [error localizedDescription];
-                self.queryView.alertText = errorMsg;
+                // TODO: Sharker fix OCR Error in tips
+//                [self.tipsCell updateTipsCellType:EZTipsCellTypeWordSelectionOCR];
```

Comment:
The OCR error display can be handled in another PR.

Reply:
okk